src/app/services/myinfo/myinfo.util.ts

```diff
@@ -0,0 +1,199 @@
+import {
+  CATEGORICAL_DATA_DICT,
+  IMyInfoAddr,
+  IMyInfoPhoneNo,
+  IPersonBasic,
+  MyInfoSource,
+} from '@opengovsg/myinfo-gov-client'
+import bcrypt from 'bcrypt'
+import crypto from 'crypto'
+import { get } from 'lodash'
+import mongoose from 'mongoose'
+
+import { sessionSecret } from '../../../config/config'
+import { IHashes, IMyInfoHashSchema, MyInfoAttribute } from '../../../types'
+import getMyInfoHashModel from '../../models/myinfo_hash.server.model'
+
+import { IPossiblyPrefilledField, MyInfoHashPromises } from './myinfo.types'
+
+const HASH_SALT_ROUNDS = 10
+const MyInfoHash = getMyInfoHashModel(mongoose)
+
+/**
+ * Formats MyInfo attribute as phone number
+ * @param phone
+ * @example +65 98654321
+ * @returns Phone number if phone.nbr exists. Else return empty string.
+ */
+const formatPhoneNumber = (phone: IMyInfoPhoneNo): string => {
+  if (!phone || !phone.nbr) return ''
+
+  return phone.prefix && phone.code && phone.nbr
```

Comment:
is is necessary to check phone.nbr again since this is already checked in L29?

Reply:
Good question! Probably not - I just copied the existing code here, but we can remove that check